docs/user/guides/subprojects.rst

```diff
@@ -0,0 +1,54 @@
+How to manage subprojects
+=========================
+
+This guide shows you how to manage subprojects,
+which are a way to host multiple projects under a "main project".
+
+.. seealso::
+
+   :doc:`/subprojects`
+      Read more about what the subproject feature can do and how it works.
+
+Adding a subproject
+-------------------
+
+In the admin dashboard for your project, select :guilabel:`Subprojects` from the menu.
+From this page you can add a subproject by choosing the subproject in the :guilabel:`Subproject` dropdown
+and typing an alias in the :guilabel:`Alias` field.
+
+Immediately after adding the subproject, it will be visible from the link displayed in the updated list of subprojects.
+
+.. image:: /img/screenshot_subprojects_list.png
+    :alt: Screenshot of a subproject immediately visible in the list after creation
+
+
+.. note::
+
+   * On |org_brand|, you need to be maintainer of a subproject in order to choose it from your main project.
+   * On |com_brand|, you need to have admin access to the subproject in order to choose it from your main project.
```

Comment:
I propose to not use bullet list for this differentiation. We can use the following approach, which I think it renders better and it's easier to read:

```suggestion
.. note::

   |org_brand|
      You need to be *maintainer* of a subproject in order to choose it from your main project.

   |com_brand|
      You need to have *admin access* to the subproject in order to choose it from your main project.
```

### Example

![Screenshot_2022-12-21_17-08-34](https://user-images.githubusercontent.com/244656/208950421-79a54ffb-e589-41bb-b20d-d5535efe9cbb.png)